add ES6 windows scan support

TerraZero opened the pull request.

Commit:
add ES6 windows scan support

jaumard commented:
Hello @TerraZero and thanks for this PR ! Can you add it also for the ES5 parser `lib/parseres5.js` ?

Commit:
fix comment regex for windows

Commit:
fix comment regex for windows

fix regex for next line in windows

fix regex for windows ES6

fix comment regex for windows

fix regex for next line in windows

only to test it in windows

fix regex for windows ES6

Commit:
fix regex for next line in windows

Commit:
for tests

Commit:
fix regex for windows ES6

Commit:
remove unwanted added test script

TerraZero commented:
Hey @jaumard I've changed the parser for ES5 and ES6. All tests have been successful on both Windows and Ubuntu. Maybe you can find a better solution to remove all the `\r` than using `.split('\r').join(")`

Commit:
Merge branch 'windows-support' of https://github.com/TerraZero/tzero-annotations into windows-support

Commit:
remove unwanted test script
